taworks/taform/templates/taform/home.html

```diff
@@ -0,0 +1,38 @@
+{% if user.is_authenticated %}
```

Comment:
Can you do the user.is_authenticated check as part of views.py?

for example:
def home(request):
    if not request.user.is_authenticated:
        return render(request, 'myapp/login_error.html')
    else:
        return render(request, 'taform/home.html')